### PR TITLE
add Staketab chains data

### PR DESCRIPTION
**Pull request by icohigh**

### CODE REVIEW

File: testnets/upticktestnet/chain.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/UptickNetwork/uptick",
+    "recommended_version": "v0.2.6",
+    "compatible_versions": [
+      "v0.2.6"
+    ],
+    "cosmos_sdk_version": "0.46.5",
+    "tendermint_version": "0.34.23",
```
Comment (JeremyParish69) [inline testnets/upticktestnet/chain.json:36]: oh, tendermin_version was changed for the consensus object (allows for tendermint of cometbft).
Also, if it's not too much trouble, could you please add this version data into the versions array?

Comment (icohigh) [reply testnets/upticktestnet/chain.json]: Check the fix

**Comment by JeremyParish69** [comment]
Please see addition of version object:
https://github.com/Staketab/chain-registry/pull/2

**Comment by JeremyParish69** [comment]
Thanks for adding that. Unfortunately, looks like there are conflicts now that needs to be resolved.

**Comment by icohigh** [comment]
> Thanks for adding that. Unfortunately, looks like there are conflicts now that needs to be resolved.

Check now, please

**Pull request merged**